DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/wallet/paypal/activate-paypal.md

4. Click on _Grant API permission_
5. Enter `paypal_api1.multisafepay.com` in the Third-Party Permission Username field. Click on _Lookup_.

> {{< alert-notice >}} __Please read:__ If any of these steps are inaccurate, send an email to: <integration@multisafepay.com> {{< /alert-notice >}} 

Reviewer: ```suggestion
{{< alert-notice >}} For support or if any steps are inaccurate, email the Integration Team at <integration@multisafepay.com> {{< /alert-notice >}} 
```

Your account is now configured and you are ready to accept MultiSafepay payments. We strongly recommend you perform one or more test transactions in order to ensure everything is working correctly.

> If PayPal is not visible as a payment method in your MultiSafepay Control account, contact our Integration Team at <integration@multisafepay.com> to assist you further.

Reviewer: ```suggestion
If PayPal is not visible as a payment method in your MultiSafepay Control account, email the Integration Team at <integration@multisafepay.com> 
```
